{"name": "test_eval_2[2-1-1]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 35 == 30", "trace": "test_input = '2-1', expected = 1\n\n    @pytest.mark.parametrize('test_input,expected',[('3+5',8),\n    ('2-1',1),('7*5',30)])\n    def test_eval_2(test_input,expected):\n        assert eval(test_input)==expected\n>       assert 35 == 30                     # 提示把30改成35\nE       assert 35 == 30\n\ntest_demo.py:15: AssertionError"}, "parameters": [{"name": "test_input", "value": "'2-1'"}, {"name": "expected", "value": "1"}], "start": 1663130373623, "stop": 1663130373623, "uuid": "1d2bc350-cceb-430e-abfa-1975491846fb", "historyId": "45f96a91b2df48fdb3f86d5bb280747e", "testCaseId": "4a96b9bee4b19b1cdeac3c287387c9ed", "fullName": "test_case.demo_case.stu_demo.test_demo#test_eval_2", "labels": [{"name": "parentSuite", "value": "test_case.demo_case.stu_demo"}, {"name": "suite", "value": "test_demo"}, {"name": "host", "value": "我不帅"}, {"name": "thread", "value": "10500-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.demo_case.stu_demo.test_demo"}]}